File: CC++/Assignments/Assignment 6/gcd.c
/*GCD*/
#include <stdio.h>
main()
{

    int a, b, i;
    printf("Enter the 2 no : ");
    scanf("%d%d", &a, &b);
    for (i = a > b ? b : a; i > 0; i--)
        if (a % i == 0 && b % i == 0)
            break;

    printf("GCD = %d\n", i);
}

File: Java/Assignment 1/GCD.java
import java.util.Scanner;

public class GCD {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter Two numbers : ");
        int a = in.nextInt();
        int b = in.nextInt();

        System.out.println("The GCD is :" + gcd(Math.max(a, b), Math.min(a, b)));
        in.close();
    }

    static int gcd(int a, int b) {
        if (b == 1)
            return a;
        return gcd(b, a % b);
    }
}
